packages/ibm-products/src/components/Datagrid/utils/DatagridPagination.js

```diff
@@ -18,6 +18,7 @@ export const DatagridPagination = ({ state, setPageSize, gotoPage, rows }) => {
 
   return (
     <Pagination
+      {...state}
```

Comment:
This is likely not what you meant to do because this attaches all of the datagrid state to the Carbon `Pagination` component which ends up applying most of it as attributes, see below:

<img width="875" alt="image" src="https://github.com/carbon-design-system/ibm-products/assets/10215203/14c3de03-7f77-497d-a122-0df8143c5bd1">

Reply:
ohh interesting. thanks for the heads up!